packages/components/src/components/domainproperties/models.ts

```diff
@@ -1159,6 +1159,13 @@ export class DomainException extends Record({
                 errors = DomainFieldError.fromJS(rawModel.errors, severityLevel);
             }
 
+            let severity = severityLevel;
+            let hasOnlyWarnings = errors.find(error => error.severity === SEVERITY_LEVEL_WARN);
```

Comment:
Wouldn't this "find" just get the first error where the severity is warn? I think you might be able to use something like `errors.every()`.

Reply:
Also, looking at DomainFieldError.fromJS(), doesn't it just set the error.severity based on the passed in second parameter? If so, wouldn't the hasOnlyWarnings be true if the severifyLevel param here in create was warn? Or maybe I am reading this wrong because I would have expected the severity to be coming back from the server.